components/ComponentAbstract.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace RainLab\Blog\Components;
```

Comment:
Formatting still incorrect

Reply:
Also, this should be in Classes, not Components

Reply:
Done. Also fixed one issue.